### DISCUSSION TITLE
🙏 How do I get my `pages/about.js` to export as `about/index.html` instead of `about.html` Thanks!

### DISCUSSION THREAD

**Discussion by jpmcglone** [vercel/next.js#15107] [Help]
🙏 How do I get my `pages/about.js` to export as `about/index.html` instead of `about.html` 

Thanks!

<img width="168" alt="Screen Shot 2020-07-12 at 5 38 05 PM" src="https://user-images.githubusercontent.com/4998416/87257159-7fea6f00-c466-11ea-9490-6e912f7474eb.png">

You'll see I'm also having this issue with `tags`, and you can't see it, but I have the issue with `tag/*` as well. Oh, and `post/*` -- to visit the site directly, you need to have `.html` at the end, and that's horrible! Any help would be GREATLY appreciated.

**Comment by illuminist** [comment]
Is making route `pages/about/index.js` works for you?

**Reply by jpmcglone** [reply]
Unfortunately, making `pages/about/index.js` still generates `about.html` haha

**Comment by Timer** [comment]
https://nextjs.org/docs/api-reference/next.config.js/exportPathMap#adding-a-trailing-slash
